uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/wizard/NewFuseIgniteExtensionProjectDependenciesPage.java

```diff
@@ -0,0 +1,98 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Red Hat, Inc. 
+ * Distributed under license by Red Hat, Inc. All rights reserved. 
+ * This program is made available under the terms of the 
+ * Eclipse Public License v1.0 which accompanies this distribution, 
+ * and is available at http://www.eclipse.org/legal/epl-v10.html 
+ * 
+ * Contributors: 
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.fuse.reddeer.wizard;
+
+import java.util.List;
+
+import org.eclipse.reddeer.core.reference.ReferencedComposite;
+import org.eclipse.reddeer.jface.wizard.WizardPage;
+import org.eclipse.reddeer.swt.impl.button.PushButton;
+import org.eclipse.reddeer.swt.impl.combo.LabeledCombo;
+
+/**
+ * Represents the second page of "New Fuse Ignite Extension Project" wizard
+ * 
+ * @author tsedmik
+ */
+public class NewFuseIgniteExtensionProjectDependenciesPage extends WizardPage {
+
+	public NewFuseIgniteExtensionProjectDependenciesPage(ReferencedComposite referencedComposite) {
+		super(referencedComposite);
+	}
+
+	public LabeledCombo getSpringBootVersionCMB() {
+		return new LabeledCombo("Spring Boot Version");
```

Comment:
Please reuse the referenced composite, i.e.
```
new LabeledCombo(this, "Spring Boot Version");
```
Also apply the above rule wherever you can.

Reply:
done ;-)